src/Core/AnimationPlayer.js

```diff
@@ -55,6 +55,7 @@ class AnimationPlayer extends THREE.EventDispatcher {
         this.duration = 0;
         this.state = PLAYER_STATE.STOP;
         this.waitTimer = null;
+        this.callback = () => {};
```

Comment:
move in one commit

Reply:
I separated this part in another commit.